assets/operator/ocp-x86_64/dotnet/imagestreams/dotnet-rhel.json

```diff
@@ -43,7 +89,7 @@
 					"openshift.io/display-name": ".NET Core 3.1 (UBI 8)",
 					"sampleContextDir": "app",
 					"sampleRef": "dotnetcore-3.1",
-					"sampleRepo": "https://github.com/redhat-developer/s2i-dotnetcore-ex.git",
+					"sampleRepo": "https://github.com/redhat-developer/s2i-dotnetcore-ex",
 					"supports": "dotnet:3.1,dotnet",
 					"tags": "builder,.net,dotnet,dotnetcore,dotnet31",
 					"version": "3.1"
```

Comment:
3.1-ubi8 tag is not removed from this pr

3.1-ubi8
tagged from registry.access.redhat.com/ubi8/dotnet-31:3.1

Reply:
good catch @xiuwang - thanks !

@rolfedh I'm updating the removed list in this PR accordingly